CHANGELOG

```diff
@@ -27,6 +27,7 @@ Node Versions:
 
 NOTICE: Restart wiseService before capture when upgrading
 NOTICE: Cross-cluster Shortcuts require you to not restart all your viewers at once after upgrading
+NOTICE: Create a parliament config file before upgrading (see arkime.com/settings#parliament)
```

Comment:
@awick I'm going to great a parliament section on the settings page. Do you think this is a good NOTICE or is the BREAKING below enough?

Reply:
I think its fine, the FAQ entry will probably be more important